src/components/atoms/popUpHeader.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { View, Text } from 'react-native';
+import ImageBeer from './imageBeer';
+
+import AtomStyle from '../style/atomsStyle';
+
+const PopUpHeader = ({ name, price, url }) => {
+    const priceStr = String(price / 100);
+    // String Contains Decimal
+    if (priceStr.includes('.')) {
+        if (priceStr.split('.')[1].length < 2) {
+            var priceShown = priceStr.split('.')[0] + ',' + priceStr.split('.')[1] + '0 €';
+        } else {
+            var priceShown = priceStr.split('.')[0] + ',' + priceStr.split('.')[1] + '€';
+        }
+    } else {
+        var priceShown = priceStr + '€';
+    }
```

Comment:
Dans un premier temps, je pense qu'il est mieux de faire une fonction au lieu d'un traitement brutal. Genre:
const blabla = (price) => return ...

Et dans un second temps, j'ai trouvé une manière plus propre de le faire 

https://stackoverflow.com/questions/6134039/format-number-to-always-show-2-decimal-places